tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -992,32 +992,31 @@ void test_spec_replacement_field() {
     test_string_specs<charT>();
 }
 template <class charT, class... Args>
-void test_size_helper_impl(
-    const size_t expected_size, const _Basic_format_string<charT, Args...> fmt, const Args&... args) {
-    assert(formatted_size(fmt, args...) == expected_size);
-    assert(formatted_size(locale::classic(), fmt, args...) == expected_size);
+void test_size_helper_impl(const size_t expected_size, const _Basic_format_string<charT, Args...> fmt, Args&&... args) {
+    assert(formatted_size(fmt, forward<Args>(args)...) == expected_size);
+    assert(formatted_size(locale::classic(), fmt, forward<Args>(args)...) == expected_size);
 
     const auto signed_size = static_cast<ptrdiff_t>(expected_size);
     basic_string<charT> str;
     {
         str.resize(expected_size);
-        const auto res = format_to_n(str.begin(), signed_size, fmt, args...);
+        const auto res = format_to_n(str.begin(), signed_size, fmt, forward<Args>(args)...);

```

Comment:
`forward` is extraneous here - no formatter is or ever should be sensitive to value category. (Also on 1011 and 1019.)

Reply:
the forward isn't for the formatter, but because all types must match exactly for `fmt` to be a valid parameter to `format_to_n`

Reply:
Ah - the value category is encoded in `fmt`? We'll worry about this in a year or so when we're making a throughput pass.